tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -0,0 +1,60 @@
+import logging
+import re
+
+import pytest
+import random
+from ocs_ci.framework.testlib import (
+    tier4, ManageTest, aws_platform_required, ignore_leftovers, ipi_deployment_required)
+from ocs_ci.ocs import machine
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import ocp
+from ocs_ci.ocs import node
+from tests.sanity_helpers import Sanity
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@ignore_leftovers
+@aws_platform_required
+@ipi_deployment_required
+class Testknip894(ManageTest):
+    """
+    Knip-678 Node Replacement
+    """
+
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+        """
+        self.sanity_helpers = Sanity()
+
+    def test_nodereplacement_proactive(self, pvc_factory, pod_factory):
+        """
+        Knip-894 Node Replacement proactive
+        """
+        self.sanity_helpers.create_resources(pvc_factory, pod_factory)
```

Comment:
Will IOs will be running till the end of test? how is this made sure?....

Reply:
No IO will not be running till end as its not running in background

Reply:
We need IOs to be running throughout the test to make sure that replacement did not impact the IOs (unexpectedly)

Reply:
will check if we have any function running io in bg and update script